src/components/QuizMultiChoice.vue

```diff
@@ -1,34 +1,46 @@
 <template>
-	<div class="quiz-multiple">
-		<div v-for="answer in choices">
-			<input type="checkbox" :id="answer.text" name="answer"
-				:value="answer.text" v-model="inputSelected" >
-			</input>
-			<label class="question-answer" :for="answer.text" :class="{'answer-selected': inputSelected.includes(answer.text)}">{{ answer.text }}</label>
-   	</div>
-	</div>     
+  <div class="quiz-multiple">
+    <div v-for="answer in choices">
+      <input type="checkbox" :id="answer.text" name="answer"
+        :value="answer.text" v-model="inputSelected" >
+      </input>
```

Comment:
This `input` element can be a self-closing tag like this `<input />`. Some explanations: https://hackernoon.com/to-close-or-not-to-close-4365d24853ff